modules/datastore/modules/datastore_mysql_import/tests/data/multiple_empty_rows.csv

```diff
@@ -0,0 +1,6 @@
+"id","name","score"
+1,"Yessenia",67.1
+,,
+2,"Roberto",2.2
+'','',''
```

Comment:
Tests are failing due to this line with the empty quotes.   I am looking into why.